protos/google/api/annotations.proto

```diff
@@ -0,0 +1,31 @@
+// Copyright 2015 Google LLC
```

Comment:
please move that to base

Reply:
Can't do: this is how it is imported
⇒ it needs to be in `google/api/...`

Somehow (don't ask me why) this is ["best practice"](https://github.com/grpc-ecosystem/grpc-gateway/issues/1065#issuecomment-544241612)
=> The files we need are added from https://github.com/googleapis/googleapis

here is the documentation on this https://grpc-ecosystem.github.io/grpc-gateway/docs/tutorials/adding_annotations/#using-protoc

After a bit of more careful documentation reading, I have removed a few api files

Reply:
I meant the whole proto folder :D or is that also not possible?